modules/src/ics02_client/msgs/create_client.rs

```diff
@@ -19,7 +19,7 @@ use crate::ics02_client::error::{Error, Kind};
 use crate::ics24_host::identifier::ClientId;
 use crate::tx_msg::Msg;
 
-const TYPE_MSG_CREATE_CLIENT: &str = "create_client";
+pub const TYPE_URL: &str = "/ibc.core.client.v1.MsgCreateClient";
```

Comment:
FYI: Introduced these public constants in all messages to be able to use them in pattern `match`ing, which is useful in the ICS26 handler:

https://github.com/informalsystems/ibc-rs/blob/a82f53a1b58dc868f3e865d34313bf995e9d7291/modules/src/ics26_routing/handler.rs#L23-L26